lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -228,14 +222,14 @@ private JourneyResponse getJourneyResponse(
                             clientOAuthSessionItem.getUserId(), govukSigninJourneyId, ipAddress);
 
             var ciScoringCheckResponse = checkForCIScoringFailure(contraIndicators);
-
             Optional<Boolean> reproveIdentity =
                     Optional.ofNullable(clientOAuthSessionItem.getReproveIdentity());
 
-            if (reproveIdentity.orElse(false)) {
-                return buildForceResetResponse(ciScoringCheckResponse.orElse(null));
-            } else if (configService.enabled(RESET_IDENTITY)) {
-                return buildForceGpg45ResetResponse(ciScoringCheckResponse.orElse(null));
+            if (reproveIdentity.orElse(false) || configService.enabled(RESET_IDENTITY)) {
+                LOGGER.info(
+                        LogHelper.buildLogMessage(
+                                "resetIdentity flag is enabled, reset users identity."));
+                return JOURNEY_RESET_SESSION_IDENTITY;
```

Comment:
This doesn't actually need to reset the session identity, it can link straight through to `NEW_P2_IDENTITY` / `REPROVE_IDENTITY`.

I'd probably call the event `/journey/reprove-identity` or similar

Reply:
I think this comment still applies :)